nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -403,6 +403,8 @@
 
 - `gerbera` now has wavpack support.
 
+- A toggle has been added under the `users.users.<name>.enable` to allow toggling individual users conditionally. If set to false, the user account will not be created.
```

Comment:
```suggestion
- A toggle has been added under `users.users.<name>.enable` to allow toggling individual users conditionally. If set to false, the user account will not be created.
```

Reply:
This was meant to be `the ... option`. Whoops.

Fixing now.